### PR TITLE
Added em-hrs-ingestor to em namespace

### PR DESCRIPTION
**Pull request by yogesh-hullatti**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-3650

### Change description ###
Added em-hrs-ingestor to em namespace

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by crimesp — APPROVED** [review]

File: k8s/namespaces/em/kustomization.yaml [hunk 1]
```diff
@@ -7,4 +7,5 @@ bases:
 - em-anno/em-anno.yaml
 - em-stitching/em-stitching.yaml
 - em-ccd-orchestrator/em-ccd-orchestrator.yaml
-- em-hrs-api/em-hrs-api.yaml
\ No newline at end of file
+- em-hrs-api/em-hrs-api.yaml
+- em-hrs-ingestor/em-hrs-ingestor.yaml
```
Comment (timja) [inline k8s/namespaces/em/kustomization.yaml:11]: this adds you to all environments and will make the app go live? is that intentional?

Comment (yogesh-hullatti) [reply k8s/namespaces/em/kustomization.yaml:11]: Yes.

Comment (timja) [reply k8s/namespaces/em/kustomization.yaml:11]: have you got an ATO / approval to deploy?

Comment (yogesh-hullatti) [reply k8s/namespaces/em/kustomization.yaml:11]: No. Checking this with my DM

**Review by crimesp — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**